scripts/import_document.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility app for importing documents from Office Open XML file formats.
+"""
+
+from __future__ import print_function
+import sys
+import os.path
+import mimetypes
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaFileUpload
+from google import auth
+
+mimetypes.init()
+
+GOOGLE_SHEETS = 'application/vnd.google-apps.spreadsheet'
+GOOGLE_DOCS = 'application/vnd.google-apps.document'
+GOOGLE_SLIDES = 'application/vnd.google-apps.presentation'
+
+TYPE_MAP = {
```

Comment:
It could be though not sure it's worth it. There's also the risk that it could break in the future. Each mime type in 'importFormats' maps to an array of potential formats. Currently for these mappings it's all 1-to-1, but there's no guarantee it stays that way.

Reply:
SGTM